Python/Distance-Calculator-App/main.py

```diff
@@ -0,0 +1,19 @@
+from geopy.geocoders import Nominatim
+from geopy import distance
```

Comment:
It's not present in the `requirements.txt`

Reply:
> It's not present in the `requirements.txt`

geopy module present in requirements.txt. This is the only one module to be installed to run file.